packages/blade/src/components/Input/OTPInput/OTPInput.stories.tsx

```diff
@@ -243,3 +245,37 @@ const OTPInputControlledTemplate: ComponentStory<typeof OTPInputComponent> = ()
   return <OTPInput label="Enter OTP" value="123456" name="otp" />;
 };
 export const OTPInputControlled = OTPInputControlledTemplate.bind({});
+
+export const inputRef: ComponentStory<typeof OTPInputComponent> = () => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const inputRef = React.useRef<HTMLInputElement[]>([]);
+
+  return (
+    <Box gap="spacing.3" display="flex" alignItems="end">
+      <OTPInputComponent
+        ref={inputRef}
+        label="Enter OTP"
+        name="otp"
+        onChange={({ name, value }): void => console.log({ name, value })}
+      />
+      <Button
+        onClick={() => {
+          console.log(inputRef);
+          inputRef?.current[1].focus();
+        }}
+      >
+        Click to focus the OTPInput
+      </Button>
```

Comment:
For more interactivity, I feel we can make this a dropdown and provide options as Focus 1st input, Focus 2nd input, ...

Reply:
Focusing after dropdown state change might cause issues since dropdown's focus will try to go to the target and we will try to move the focus to the OTP input.

Reply:
Okay, added a dropdown with an explicit button to focus on.